### ISSUE TITLE
Import form node_modules into dist folder

### ISSUE THREAD

**Issue by zimexco** [angular/angular-cli#140]
Hi! I want to use bootstrap v4 with my project generated by angular-cli, but I can't find a way to import the bootstrap folder downloaded with npm inside dist/vendor .
How can I do?

Thank's

**Comment by Gillespie59** [comment]
I think you can not, because the files imported in the vendor directory is hard-coded. Cf this file https://github.com/angular/angular-cli/blob/master/lib/broccoli/angular2-app.js

Am I wrong ? 

Maybe we can do the same thing as wiredep ? read the main property of each NPM modules, and copy/paste these files in the vendor dir ?

**Comment by intellix** [comment]
If you want to quickly solve this, have this ember-cli-build.js:

```js
/* global require, module */

var Angular2App = require('angular-cli/lib/broccoli/angular2-app');
var Funnel = require('broccoli-funnel');
var mergeTrees = require('broccoli-merge-trees');

module.exports = function(defaults) {
    var app = new Angular2App(defaults);
    var app = app.toTree();

    var vendors = new Funnel('node_modules/bootstrap/scss', {
        destDir: 'vendor/bootstrap/scss'
    });

    return mergeTrees([app, vendors], { overwrite: true });
}
```

**Comment by cironunes** [comment]
Closed by acaaa3ea4bacd7b85d0e6c5ad2f3e7b5e48a1498 and 2efc9a150fe84a38d479ed062e6454f3e3a14970
